ldi-core/ldes-client/tree-node-relations-fetcher/src/test/java/ldes/client/treenoderelationsfetcher/WireMockConfig.java

```diff
@@ -0,0 +1,30 @@
+package ldes.client.treenoderelationsfetcher;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+
+public class WireMockConfig {
+
+	private static final int WIREMOCK_PORT = 10101;
+	public static WireMockServer wireMockServer;
+
+	@Before
+	public static void setupWireMock() {
+		wireMockServer = new WireMockServer(
+				WireMockConfiguration.options().port(WIREMOCK_PORT));
+		System.out.println("----- Starting Wiremock Server -----");
+		if (!wireMockServer.isRunning()) {
+			wireMockServer.start();
+		}
+	}
+
+	@After
+	public static void tearDownWireMock() {
+		System.out.println("----- Stopping Wiremock Server -----");
+		wireMockServer.stop();
+	}
```

Comment:
starting to think we should move this to common.. found this already twice in 2 other modules